react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -38,5 +38,9 @@ class Board extends Component {
   
 }
 
+Board.propTypes = {
+  squares: PropTypes.array.isRequired,
```

Comment:
Here you are defining an array, but not specifying what kind of data the array contains. A good practice is to use `PropTypes.arrayOf(<Type>)` to represent it.

For example, an array of numbers would be: `PropTypes.arrayOf(PropTypes.number)`

In your case, you have an array of `Square`. Therefore, you can do 2 things:
1 - Use `PropTypes.shape()`:
 ```
PropTypes.arrayOf(PropTypes.shape({ 
    value: PropTypes.string,
    onClick: PropTypes.func.isRequired
  });
```
2 - create a `propTypes.js` file, where you can define `squarePropType` using `shape()` and use it here:
```
export const squarePropType = PropTypes.shape({
         value: PropTypes.string,
        onClick: PropTypes.func.isRequired,
    });

..... in your file
import { quarePropTypes } from './propTypes.js';
...
Board.propTypes = {
    squares: PropTypes.arrayOf(squarePropTypes),
    ...
}```

Reply:
i dont know if its ok, BUT if i write :
`Board.propTypes = {`
`  squares: PropTypes.arrayOf(`**Square**`).isRequired,`
`  onClick: PropTypes.func.isRequired,`
`};`

the console doesn't send any error, so its "ok".
Also other question, its ok if i do?? :
`Board.propTypes = {`
`  squares: PropTypes.arrayOf(`**PropTypes.string**`).isRequired,`
`  onClick: PropTypes.func.isRequired,`
`};`